webapp/sources/ldap-inventory/inventory-fusion/src/main/scala/com/normation/inventory/provisioning/fusion/PreInventoryParserCheckInventoryAge.scala

```diff
@@ -118,7 +118,9 @@ object PreInventoryParserCheckInventoryAge {
     val youngest = now.plus(maxAfterNow)
 
     val tooOld    = Inconsistency(s"Inventory is too old, refusing (inventory date is before '${oldest.toString}')")
-    val tooFuture = Inconsistency(s"Inventory is too far in the future, refusing (inventory date is before '${oldest.toString}')")
+    val tooFuture = Inconsistency(
+      s"Inventory is too far in the future, refusing (inventory date is before '${youngest.toString}')"
```

Comment:
```suggestion
      s"Inventory is too far in the future, refusing (inventory date is after '${youngest.toString}')"
```

Reply:
plus, I think the semantics of "youngest" and "oldest" are reversed here :exploding_head:

Reply:
it seems correct, but I'm renaming them to make things clearer - obviously there are not